tests/integration/api/test_extra_var.py

```diff
@@ -56,3 +56,27 @@ def test_retrieve_extra_var(client: APIClient):
 def test_retrieve_extra_var_not_exist(client: APIClient):
     response = client.get(f"{api_url_v1}/extra-vars/42/")
     assert response.status_code == status.HTTP_404_NOT_FOUND
+
+
+@pytest.mark.parametrize(
+    "extra_var", ["John", "John, ", "[John, 3,]", '{"name": "John" - 2 }']
+)
+@pytest.mark.django_db
+def test_extra_var_invalid_data(client: APIClient, extra_var):
+    invalid_data = {
+        "extra_var": extra_var,
+    }
+    response = client.post(f"{api_url_v1}/extra-vars/", data=invalid_data)
+    assert response.status_code == status.HTTP_400_BAD_REQUEST
```

Comment:
@Dostonbek1 Can we test what the error message looks like for validation error, this would be needed by @lgalis to display in the UI.